package.json

```diff
@@ -16,6 +16,7 @@
     "url": "git://github.com/vigetlabs/gangway.git"
   },
   "dependencies": {
+    "inflected": "1.1.6",
```

Comment:
I want this to be hyper specific. Changes to inflection should not be taken lightly :P